webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/DataSourceController.scala

```diff
@@ -149,16 +150,17 @@ class DataSourceController @Inject()(
         tuple(
           "resumableChunkNumber" -> number,
           "resumableChunkSize" -> number,
+          "resumableCurrentChunkSize" -> number,
```

Comment:
@fm3 this should be fine to add as a param as it is already sent by previous wklibs versions, correct @markbader?

Mroeover, now that I added this, your wklibs pr should hopefully be working now :+1:.

Reply:
Yes, should be fine. The old libs do send this already.
If someone lies about this number, the chunk request may wrongly pass, but finishUpload should ultimately be rejected, which seems fine to me. :shipit: